controlplane/src/core/repositories/FeatureFlagRepository.ts

```diff
@@ -403,6 +667,7 @@ export class FeatureFlagRepository {
         isFeatureGraph: subgraphs.isFeatureSubgraph,
         baseSubgraphId: featureSubgraphsToBaseSubgraphs.baseSubgraphId,
         isEventDrivenGraph: subgraphs.isEventDrivenGraph,
+        isFeatureSubgraph: subgraphs.isFeatureSubgraph,
```

Comment:
Should we go with `isFeatureSubgraph` or `isFeatureGraph` ?

Reply:
We call it Feature Subgraph, so isFeatureSubgraph